website/docs/04-roadmap.md

```diff
@@ -0,0 +1,7 @@
+# Roadmap
+
+## Q4 2023
+* 🔧 Improve RAG by deeper integration with Treesitter using custom query. This will bring LSP-lik understanding to tabby's code index.
```

Comment:
```suggestion
* 🔧 Improve RAG by deeper integration with Treesitter using custom query. This will bring LSP-like understanding to Tabby's code index.
```